helm/charts/hydra/templates/deployment.yaml

```diff
@@ -64,14 +64,14 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.hydra.customCommand }}
```

Comment:
Hi there! 
I'd prefer a slightly different format:
`Values.hydra.command` with the default value of `hydra` and `Values.hydra.args` with the default value of `[]`. This will be more verbose and allows us to drop the if/else clauses :)

Reply:
Sure, I agree with having `Values.hydra.command` to avoid the if/else clauses! Do you think is a good  idea to set `Values.hydra.args` default to `[]` when the previous version was like this: 
```
          args:
            - serve
            - all
            {{- if .Values.hydra.dev }}
            - "--dev"
            {{- end }}
            - --config
            - /etc/config/hydra.yaml
```

Reply:
Ah true! Forgot about that bit 😅.
For args i think it makes sense to leave customArgs as you proposed as that will override the whole structure. In that case we would need to make it like 

```yaml
{{- if Values.hydra.customArgs }}
args:
...
{{- else }}
          args:
            - serve
            - all
            {{- if .Values.hydra.dev }}
            - "--dev"
            {{- end }}
            - --config
            - /etc/config/hydra.yaml
{{- end }}

Reply:
I made the chages :)